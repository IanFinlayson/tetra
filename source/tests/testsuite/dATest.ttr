def main() int:
   x = [2,4,6]
   y = call(x)
   x[1] = 10
   return y[1]

def call(a [int]) [int]:
   return a
