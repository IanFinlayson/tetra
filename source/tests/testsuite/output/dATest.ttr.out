Just created x and x = [2, 4, 6]
In call() and a = [2, 4, 6]
Just created y, set y = call(x),  and y = [2, 4, 6]
Just set x[1] = 10 and x = [2, 10, 6]
and y = [2, 10, 6]

